python/ray/tune/experimental/output.py

```diff
@@ -85,6 +85,7 @@
     "trial_id",
     "experiment_tag",
     "should_checkpoint",
+    "_report_on",  # LIGHTNING_REPORT_STAGE_KEY
```

Comment:
Actually this is created by us in LightningTrainer. Yes, we should put it into the blacklist.